docs/it-manual/sre-playbook/production-database-access.md

```diff
@@ -17,7 +16,7 @@ From your CiviForm deployment repo:
 
 1. Run `./bin/run`
 1. When prompted for a command to run, enter `pgadmin`
-1. The script will attempt to detect your IP address and suggest an IP-allowlist containing it. If the web browser you will use to access pgadmin is on a different machine than the one running the script (e.g. because you are using cloudshell), add the CIDR mask for the IP address of local network instead. (Follow on-screen instructions for this).
-1. Wait for the pgadmin instance to become available
+1. The script will attempt to detect your IP address and suggest an IP-allowlist containing it. If the web browser you will use to access pgadmin is on a different machine than the one running the script, add the CIDR mask for the IP address of local network instead. (Follow on-screen instructions for this).
+1. Wait for the pgadmin instance to become available. (If the script is still polling after a minute or so, it's ok to press ctrl+c **once** to cancel polling to get the access credentials and trying visiting pgadmin in your browser)
```

Comment:
**nitpick:** I think it would be "and try visiting", but I think you can remove " and trying visiting pgadmin in your browser" since that's in the next step?

Reply:
Done